gui/advancedview.cpp

```diff
@@ -124,6 +124,11 @@ void AdvancedView::update(Cluster cluster)
 #endif
     pauseButton->setText(getPauseLabel(isPaused));
     startButton->setText(getStartLabel(isRunning));
+    QString startToolTip = "";
+    if (isRunning) {
+        startToolTip = "Restart the Kubernetes API";
```

Comment:
Restarts the already started minikube that might refresh the config

Reply:
Reworded to `Restart an already running minikube instance to pickup config changes.`